package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@codahq/packs-sdk",
-  "version": "1.3.6",
+  "version": "1.3.7",
```

Comment:
Is this version bump that workaround for the `coda` repo? I seem to recall that the changelog tool doesn't like it when we skip versions.

Reply:
Yeah it was for that workaround. Let me try again tomorrow without and see if it works ok.

Even with it I had trouble getting VSCode to find my new stuff, and wound up restarting the TS Server manually. So maybe that workaround would work for the version issue too.